BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -181,13 +181,30 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
   // MARK: -
   
   func updatePendingContributionsState() {
-    if disclaimerLabels.isEmpty {
-      rewardsSummaryView.disclaimerView = nil
-    } else {
-      rewardsSummaryView.disclaimerView = WalletDisclaimerView().then {
-        $0.labels = disclaimerLabels
+    state.ledger.pendingContributionsTotal(completion: { amount in
+      let labels = self.disclaimerLabels(for: amount)
+      if labels.isEmpty {
+        self.rewardsSummaryView.disclaimerView = nil
+      } else {
+        self.rewardsSummaryView.disclaimerView = WalletDisclaimerView().then {
+          $0.labels = labels
+          $0.labels.forEach {
+            $0.onLinkedTapped = { [weak self] link in
```

Comment:
this `$0` refers to `labels` or to `WalletDisclaimerView`? idk, maybe name one of the params to avoid confusion